conf/chi.conf.defaults

```diff
@@ -78,3 +78,10 @@ reconnect=60
 #
 # Amount of time from the current time to expire an entry
 expires_in=1d
+
+[storage person_lookup]
+#
+# storage person_lookup.expires_in
+#
+# Amount of time from the current time to expire an entry
+expires_in=1h
```

Comment:
I'd even go with 24 hours and maybe even more, its not like your lastname will change every hour.

Unless you're called Zammit/Zayme/Zaymitte/ZiCEO